stdlib/public/core/StringGuts.swift

```diff
@@ -150,6 +150,11 @@ extension _StringGuts {
     defer { _fixLifetime(self) }
     return try f(_object.fastUTF8)
   }
+  
+  @inlinable
+  internal var fastUTF8: UnsafePointer<UInt8>? {
+    return _object.fastUTF8.baseAddress
+  }
```

Comment:
This should return something that's non-optional. There will be an assertion failure inside _StringObject if this isn't capable of providing fast-UTF8, and optional implies this can be used as a query. I would also have "Start" at the end of the name, e.g. see `_StringObject.nativeUTF8Start` and `_StringObject.sharedUTF8Start`.

Reply:
But, if this is only called by the storage class, then we should just use the class's pointer and can remove this.

Reply:
Sounds good!